Description: Fixes file location constants to more updated/organized variants. 
  	~/.fonts.conf -> ~/.config/fontconfig/fonts.conf
		~/.fonts/     -> ~/.local/share/fonts/
Author: i_magnific0 (i_magnific0@yahoo.com)
Notes: patch not submitted
---
 src/constants.py.in |    4 ++--
 1 file changed, 2 insertion(+), 2 deletions(-)
 
--- font-manager.orig/src/constants.py.in
+++ font-manager/src/constants.py
@@ -75,11 +75,11 @@
 
 HOME                        =   os.getenv('HOME')
 USER                        =   os.getenv('USER')
-USER_FONT_DIR               =   join(HOME, '.fonts')
+USER_FONT_DIR               =   join(glib.get_user_data_dir(), 'fonts')
 COMPAT_COLLECTIONS          =   join(CONFIG_DIR, 'fontgroups.xml')
 USER_FONT_COLLECTIONS       =   join(DATA_DIR, 'Collections.xml')
 USER_FONT_COLLECTIONS_BAK   =   join(DATA_DIR, 'Collections.xml.bak')
-USER_FONT_CONFIG            =   join(HOME, '.fonts.conf')
+USER_FONT_CONFIG            =   join(CONFIG_DIR, 'fontconfig/fonts.conf')
 USER_FONT_CONFIG_DIR        =   join(APP_CONFIG_DIR, 'conf.d')
 USER_FONT_CONFIG_DIRS       =   join(APP_CONFIG_DIR, 'directories.conf')
 USER_FONT_CONFIG_RENDER     =   join(APP_CONFIG_DIR, 'local.conf')
